.github/actions/conditional/conditions

```diff
@@ -0,0 +1,14 @@
+# File patterns used to decide what workflows/jobs to execute for a given PR
+#
+# To test a pattern run '.github/actions/conditional/conditional.sh <remote name> <branch>'
+# Alternatively, run 'git diff <remote name>/<branch> <pattern>' (from the root directory)
+#
+# Note: The file needs to end with an empty line, otherwise last pattern is not picked up
```

Comment:
Dunno

Reply:
Suggestions would be welcome, including changes to the bash script to remove the need for this. It works for me for now though.

Reply:
If you put this in a `.editorconfig` file in the root of the project [most IDEs](https://editorconfig.org/#pre-installed) will enforce it.
```
[.github/actions/conditional/conditions]
end_of_line = lf
insert_final_newline = true
```

Reply:
The `insert_final_newline` is the essential bit here. I'd suggest not to set `end_of_line` as one would otherwise see line ending changes on Windows machines unless you set up Git to use only LF with `.gitattributes` setting `* text=auto eol=lf` (see https://www.ahus1.de/post/dotfiles-for-crossplatform-projects)

Reply:
Added

Reply:
Removed end_of_line